plugins/periskop-backend/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@backstage/plugin-periskop-backend",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": false,
+  "homepage": "https://periskop.io",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "start": "backstage-cli backend:dev",
+    "build": "backstage-cli backend:build",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^0.10.8",
```

Comment:
These dependencies need to be updated as they've been bumped since this work was started. I believe that's the source of the CI errors

Reply:
Thanks, will give it a try.